feature/search/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/search/SearchViewModel.kt

```diff
@@ -57,7 +57,9 @@ class SearchViewModel @Inject constructor(
                     flowOf(SearchResultUiState.SearchNotReady)
                 } else {
                     searchQuery.flatMapLatest { query ->
-                        if (query.length < SEARCH_QUERY_MIN_LENGTH) {
+                        if ((query.length < SEARCH_QUERY_MIN_LENGTH)
+                                .or(query.isBlank())
+                        ) {
```

Comment:
Don't over-use Kotlin extension functions where there is no need for it:

```suggestion
                        if (query.length < SEARCH_QUERY_MIN_LENGTH) || query.isBlank()) {
```

Also, what about simply trimming the query? This would take care of both cases, and make the `SEARCH_QUERY_MIN_LENGTH` more "correct"